src/cli-validator/utils/processConfiguration.js

```diff
@@ -263,6 +270,86 @@ const getFilesToIgnore = async function() {
   return filesToIgnore;
 };
 
+const validateLimits = function(limitsObject, chalk) {
+  const allowedLimits = ['warnings'];
+  const limitErrors = [];
+
+  Object.keys(limitsObject).forEach(function(key) {
+    if (!allowedLimits.includes(key)) {
+      // remove the entry and notify the user
+      delete limitsObject[key];
+      limitErrors.push({
+        message: `"${key}" limit not supported. This value will be ignored.`,
+        correction: `Valid limits for .thresholdrc are: ${allowedLimits.join(
+          ', '
+        )}.`
+      });
+    } else {
+      // valid limit option, ensure the limit given is a number
+      if (typeof limitsObject[key] !== 'number') {
+        // remove the entry and notify the user
+        delete limitsObject[key];
+        limitErrors.push({
+          message: `Value provided for ${key} limit is invalid.`,
+          correction: `${key} limit should be a number.`
+        });
+      }
+    }
+  });
+
+  // give the user corrections for .thresholdrc file
+  if (limitErrors.length) {
+    printConfigErrors(limitErrors, chalk, '.thresholdrc');
+  }
+
+  //  sets all limits options not defined by user to default
+  for (const limitOption of allowedLimits) {
+    if (!(limitOption in limitsObject)) {
+      limitsObject[limitOption] = Number.MAX_VALUE;
+    }
+  }
+
+  return limitsObject;
+};
+
+const getLimits = async function(chalk, limitsFileOverride) {
+  let limitsObject = {};
+
+  const findUpOpts = {};
+  let limitsFileName;
+
+  if (limitsFileOverride) {
+    limitsFileName = path.basename(limitsFileOverride);
+    findUpOpts.cwd = path.dirname(limitsFileOverride);
+  } else {
+    limitsFileName = '.thresholdrc';
+  }
+
+  // search up the file system for the first instance
+  // of the threshold file
+  const limitsFile = await findUp(limitsFileName, findUpOpts);
+
+  if (limitsFile !== null) {
+    try {
+      // the threshold file must be in the root folder of the project
```

Comment:
I don't think this is true - we just looked up through the whole filesystem for it. That is what we want, I think this comment is misleading